### DISCUSSION TITLE
Display Feather icons in main menu

### DISCUSSION THREAD

**Discussion by loganmarchione** [adityatelange/hugo-PaperMod#445] [Q&A] [answered]
For example, where the red circle is, I'd like to add the RSS [social icon](https://adityatelange.github.io/hugo-PaperMod/posts/papermod/papermod-icons/).
![Screenshot_20210624_130251](https://user-images.githubusercontent.com/4955337/123304164-26137580-d50e-11eb-8b78-b660160538bf.png)

I saw [this issue](https://github.com/adityatelange/hugo-PaperMod/issues/420), but as @miynat pointed out, I too would like to use the built-in icons without referencing another resource. Is there a way to use the built-in Feather icons in the main menu?

**Comment by steve-winter** [comment, marked as answer]
Not a a great answer but it works. In your config menu add:

```
    - identifier: rss
      name: RSS
      post: <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-rss"><path d="M4 11a9 9 0 0 1 9 9"></path><path d="M4 4a16 16 0 0 1 16 16"></path><circle cx="5" cy="19" r="1"></circle></svg>
      url: /index.xml
      weight: 40
```

This renders the RSS SVG from [feathericons](https://feathericons.com/?query=rss).
